src/main/java/com/google/sps/servlets/DevicesServlet.java

```diff
@@ -33,6 +33,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       return;
     }
     final String userId = currentUser.getUserId();
+    final String maxDeviceCount = request.getParameter(MAX_DEVICES_COUNT_PARAMETER_NAME);
+    final String pageToken = (String) request.getParameter(PAGE_TOKEN_PARAMETER_NAME);
```

Comment:
Maybe I am missing something, but why does only one of these need to be cast to a string?  It seems like either both should or neither should.

Reply:
yep, will fix